people/CMakeLists.txt

```diff
@@ -8,8 +8,6 @@ if(NOT VTK_FOUND)
 else()
   set(DEFAULT TRUE)
   set(REASON)
-  set(VTK_USE_FILE "${VTK_USE_FILE}" CACHE INTERNAL "VTK_USE_FILE")
```

Comment:
Is VTK not needed in people?

Reply:
The VTK_USE_FILE seems to be a lazy way to get everything to work. https://vtk.org/Wiki/VTK/Build_System_Migration

I'll try remove all of them and see where it fails.